target_chains/ethereum/sdk/solidity/PythUtils.sol

```diff
@@ -33,4 +33,38 @@ library PythUtils {
                 10 ** uint32(priceDecimals - targetDecimals);
         }
     }
+
+    /// @notice Combines two prices to get a cross-rate
```

Comment:
document all the assumptions here that are not captured within the types like prices being positive and the range that you can truly support without reverting. please also give a note that our price feeds might not look like that, we can have a feed with negative value (at some point)

Reply:
It seems you have an assumption that expo's are <= 18

Reply:
The thing i would document in particular is: under what range of exponents is this function *guaranteed not to overflow* for any set of prices. That's the safe range to use this function in.

Reply:
I would also document that you get the floor of the result in the target exponent, so small numbers can round to 0.